plugins/Live/API.php

```diff
@@ -289,6 +287,28 @@ public function getMostRecentVisitorId($idSite, $segment = false)
         return $visitor->getVisitorId();
     }
 
+    /**
+     * Returns the very first visit for the given visitorId
+     *
+     * @internal
```

Comment:
marked the new method as internal for now. Or should we make it public API? Any opinions?

Reply:
Sounds good to leave it internal as fetching first visit can be done with the existing API methods :+1: